migrate/migrator.go

```diff
@@ -268,29 +268,42 @@ func (m *Migrator) CreateGoMigration(
 }
 
 // CreateSQLMigrations creates an up and down SQL migration files.
-func (m *Migrator) CreateSQLMigrations(ctx context.Context, name string) ([]*MigrationFile, error) {
+func (m *Migrator) CreateSQLMigrations(ctx context.Context, name string, transactional bool) ([]*MigrationFile, error) {
```

Comment:
Could you please leave this method as is and instead add `CreateTxSQLMigrations`? This way we don't break existing API.

Reply:
Undid the changes and made the function CreateTxSQLMigrations